### ISSUE TITLE
Discrepancy in tech adoption

### ISSUE THREAD

**Issue by max-ostapenko** [HTTPArchive/cwv-tech-report#37]
I would expect tech report and technology data in `crawl.pages` to show the same tech adoption.

But they don't:
![Image](https://github.com/user-attachments/assets/425a62c2-3e17-4395-92f0-fad86930a1c9)

The reason most probably being we do [INNER JOIN on CrUX data](https://github.com/HTTPArchive/dataform/blob/0e395b533f23e9d35b5dee7eacb4f229120d36c2/definitions/output/core_web_vitals/technologies.js#L237) which filters out part of crawl origins.

I'm not sure if it's possible to include a full adoption, as it will not match with filters and origin percents values in CWV metrics.

A quick solution may be to mention in the chart description that it includes only origins from field data, HTTP Archive crawls have bigger coverage.

**Comment by tunetheweb** [comment]
Yes it's the "CWV Tech Report" not just the "Tech Report". It's primary use case was to show CWV data from CrUX by technology, not to report on the number of origins using a technology.

Now in theory there should be little discrepancy as we source the crawl URL list from CrUX, and then doing it back up. So other than a few sites going off line in between you'd expect to be able to get perfect joins.

However the big exception is when an origin does not get enough traffic to have segregated data (i.e. mobile and desktop) in CrUX. In that case we crawl it for both and then the join will fail. This is how we [ramped up url support in Jun 2022](https://discuss.httparchive.org/t/10x-project-exceeding-10m-and-announcing-new-capabilities/2391/1) but does lead to this discrepancy. Effectively the CWV Tech Report is as if that ramp up never happened.

But all that's fine. It's still useful to see relatively usage of technologies. Kinda l;ike limiting it to the top 9 million sites or so!

TLDR: this is working as intended.
